orderbook/src/api/get_fee_and_quote.rs

```diff
@@ -0,0 +1,238 @@
+use crate::fee::{MinFeeCalculating, MinFeeCalculationError};
+use anyhow::Result;
+use chrono::{DateTime, Utc};
+use ethcontract::{H160, U256};
+use model::h160_hexadecimal;
+use model::{order::OrderKind, u256_decimal};
+use serde::{Deserialize, Serialize};
+use shared::{
+    conversions::{big_int_to_u256, U256Ext},
+    price_estimate::{PriceEstimating, PriceEstimationError},
+};
+use std::convert::Infallible;
+use std::sync::Arc;
+use warp::{hyper::StatusCode, reply, Filter, Rejection, Reply};
+
+#[derive(Deserialize)]
+struct Query {
+    #[serde(with = "h160_hexadecimal")]
+    sell_token: H160,
```

Comment:
Why the fee uses `H160Wrapper` and here `H160`? 
Is the wrapper already H160 with serialize/deserialize impl?

Reply:
The wrapper only matter for serialization so it's not something you would see from a http standpoint. The difference is that the wrapper implement the rust trait FromStr while this serde_with only works for serialization.